<launch>
<param name="gear_rateo" value="0.027" />
<param name="wheel_radius" value="0.1575" />
<param name="apparent_baseline" value="0.97" />
<param name="x" value="0" />
<param name="y" value="0" />
<param name="th" value="0" />
<param name="method" value="0" />
<node pkg="chicago" name="odometry" type="odometry" output="screen" />
<node pkg="tf_scout" name="tf2_publisher_odom" type="tf2_scout_odom" output="screen" />
</launch>
